packages/rich-text/src/get-start-nesting-level.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+
+import { getLineIndex } from './get-line-index';
+
+/**
+ * Returns the nesting level of the list at the selection start position.
+ *
+ * @param {Object} value     The rich-text value
+ *
+ * @return {number} The nesting level, starting from 0.
+ */
+export function getStartNestingLevel( value ) {
```

Comment:
Might be good to mention `line` in this function name as it's specific to lines. Maybe `getLineNestingLevel`. Also please let's export it as unstable. :)

Reply:
Pushed 66e971d that renames the function to mention `Line`. Let me know if that's better.

> Also please let's export it as unstable. :)

Could use some guidance on this one on how to do it. Is it perhaps just a matter of prefixing the function name with `unstable_` or something else/more?

Reply:
There's some other examples in `index.js`. E.g.:

```
export { updateFormats as __unstableUpdateFormats } from './update-formats';
```

Reply:
Thanks for the pointer!

Followed the info for how to export functions as unstable. In the meantime, the code and functions in this thread have changed (latest: 4451cb3) so, I'm going to mark this thread as resolved. Thanks!